manifests/client/configure.pp

```diff
@@ -43,7 +43,7 @@
 
     file { $title:
         ensure => 'file',
-        mode    => '0644',
+        mode    => '0550',
```

Comment:
Couldn't this be 0440?  There's not need for execute on the config file.
This applies to most of the other permissions changed in this PR, not going to annotate them all.